app/config/FeatureSwitch.scala

```diff
@@ -23,7 +23,6 @@ object FeatureSwitch {
 
   val featureSwitches: Seq[FeatureSwitch] = Seq(
     AuthFeature,
-    PenaltiesEndpointsFeature,
```

Comment:
Hope you verified this penaltiesEndpoints is already true in production? Please double check prod config for me. Also attach a screenshot to the ticket for this if any like postman script or any thing that relavant. If you can blame the page and find out the reference JIRA ticket, please include that as well in confluence page before close.